templates/repo/settings/deploy_keys.tmpl

```diff
@@ -4,6 +4,45 @@
 	{{template "repo/settings/navbar" .}}
 	<div class="ui container">
 		{{template "base/alert" .}}
+		<div {{if not .HasError}}class="hide"{{end}} id="add-deploy-key-panel">
+			<h4 class="ui top attached header">
+				{{.i18n.Tr "repo.settings.add_deploy_key"}}
+				<div class="ui right">
+					<button class="ui red tiny hide-panel button" data-panel="#add-deploy-key-panel">
+						{{.i18n.Tr "cancel"}}
+					</button>
+				</div>
```

Comment:
For any other reviewer: Those are the only lines that were not copied, the cancel button.